docs/api-reference/viewport.md

```diff
@@ -37,7 +37,8 @@ Projection Matrix Options
 Geospatial Anchor Options (Optional)
   * `latitude` (Number, optional) - Center of viewport on map (alternative to center). Default to `37`.
   * `longitude` (Number, optional) - Center of viewport on map (alternative to center). Default to `-122`.
-  * `zoom` (Number, optional) - `center`. Default to `11`.
+  * `zoom` (Number, optional) - [zoom level](https://wiki.openstreetmap.org/wiki/Zoom_levels) of the viewport. Default to `11`.
+  * `focalDistance` (Number, optional) - modifier of viewport scale if `zoom` is not supplied. Default to `1`.
```

Comment:
Thanks for adding the doc. Though I wonder if users won't find this text a little vague. 

If I recall correctly, the intention was that focalDistance was supposed to be the distance in meters where objects are at `w` scale 1, but maybe it is not quite that simple?

Regardless, I think being able to explain it in those terms would make it a lot easier for users to understand how to use this.